packages/amplify_datastore/ios/Classes/FlutterModels.swift

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+ 
+import Amplify
+import Foundation
+
+// Contains the set of classes that conforms to the `Model` protocol.
+final public class FlutterModels: AmplifyModelRegistration {
+    public let version: String = "e9c358927805a236769ad01b2804e6f1"
```

Comment:
We need to figure out a mechanism to generate and pass this string in `modelSchema` or as a `configuration()` argument, cc: @fjnoyp